docs/source/storeconv.md

```diff
@@ -27,6 +20,35 @@ Useful references related to this document include:
  - The
    [`xcube.util.jsonschema`](https://github.com/dcs4cop/xcube/blob/master/xcube/util/jsonschema.py) source code
 
+## Naming Identifiers
+
+This section states what identifiers used by an xcube data store SHALL look like.
+
+In the DataStore framework, identifiers are used to denote data sources, data stores, and data accessors.
+Data store, data opener, and data writer identifiers are used to register the component as extension in a package's plugin.py.
+Identifiers MUST be unambiguous in the scope of the data store. 
+They SHOULD be unambiguous across the entirety of data stores. 
+
+There are no further restrictions for data source and data store identifiers.
+
+A data accessor identifier MUST correspond to the following scheme:
+
+`<type_specifier>:<format_identifier>:<storage_identifier>`
+
+`<type_specifier>` MUST be a valid string that specifies a data type.
+In case the type specifier has flags, the flags MUST be given in brackets, in alphabetic order, without spaces (e.g., `dataset[cube,multilevel]`).
```

Comment:
```suggestion
In case the type specifier has flags, the flags MUST be given in square brackets, in alphabetic order, separated by single commas, without spaces (e.g., `dataset[cube,multilevel]`).
```